File: Python Exams/Basic Exam Collection/Basics Online Exam - 9 and 10 March 2019/01. Tennis Equipment/01. Tennis Equipment.py
from math import ceil, floor
price_one_rocket, rocket_count, sneakers_count = [float(input()) for _ in range(3)]

all_rocket_price = price_one_rocket * rocket_count
one_sneakers_price = price_one_rocket / 6
all_sneakers_price = sneakers_count * one_sneakers_price
other_equipment = (all_rocket_price + all_sneakers_price) * 0.20
total_price = all_rocket_price + all_sneakers_price + other_equipment

print(f'Price to be paid by Djokovic {floor(total_price / 8)}')
print(f'Price to be paid by sponsors {(ceil(total_price * 7 / 8))}')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 9 and 10 March 2019/01. Tennis Equipment/01. Tennis Equipment.js
function tennisEquipment(...arg) {
    let [priceOneRocket, rocketCount, sneakersCount] = [...arg.map(Number)]

    let allRocketPrice = priceOneRocket * rocketCount
    let oneSneakersPrice = priceOneRocket / 6
    let allSneakersPrice = sneakersCount * oneSneakersPrice
    let otherEquipment = (allRocketPrice + allSneakersPrice) * 0.20
    let totalPrice = allRocketPrice + allSneakersPrice + otherEquipment

    console.log(`Price to be paid by Djokovic ${Math.floor(totalPrice / 8)}`)
    console.log(`Price to be paid by sponsors ${(Math.ceil(totalPrice * 7 / 8))}`)
}

//tennisEquipment(850, 4, 2)